### DISCUSSION TITLE
Shadow markup extension

### DISCUSSION THREAD

**Discussion by HavenDV** [CommunityToolkit/Maui#1162] [New Feature Discussions]
Makes it easier to use shadows in MAUI
Was
```xaml
<Image Source="dotnet_bot.png"
       WidthRequest="250"
       HeightRequest="310">
    <Image.Shadow>
        <Shadow Brush="Black"
                Offset="20,20"
                Radius="40"
                Opacity="0.8" />
    </Image.Shadow>
</Image>
```
Will become
```xaml
<Image Source="dotnet_bot.png"
       WidthRequest="250"
       HeightRequest="310"
       Shadow="{toolkit:Shadow Offset='20,20', Opacity=0.8, Radius=40, Brush=Black}"/>
```

**Comment by pictos** [comment]
@HavenDV you should be able to do that already... You can see more info [here](https://github.com/CommunityToolkit/Maui/discussions/679#discussioncomment-3808763)

**Reply by HavenDV** [reply]
Yes, I've already implemented this and am using it. Just trying to follow the standard flow - proposal -> issue -> PR

**Reply by pictos** [reply]
I believe you don't get it... Everything that is Bindable on Maui can be represented by markup extension, so in a lot of cases, you don't need to create a custom one... In the link there's a sample where I showed that.

**Reply by HavenDV** [reply]
Ah, now I understand, sorry. Rider doesn't seem to be aware of this feature:
<img width="495" alt="image" src="https://user-images.githubusercontent.com/3002068/234908948-d41a6462-083c-4b56-9571-a1ed648a9562.png">
I will create issue for Rider if it's not exists

**Reply by HavenDV** [reply]
Done: https://youtrack.jetbrains.com/issue/RIDER-92930/Rider-doesnt-support-MarkupExtension-syntax-for-Bindable-objects
